qemu/tests/virtio_fsd_check_info.py

```diff
@@ -28,9 +28,3 @@ def run(test, params, env):
     if vfsd_ver != qemu_ver:
         test.fail("virtiofsd version is %s which is wrong,"
                   "it should be %s" % (vfsd_ver, qemu_ver))
-    pattern_cort = r'Copyright.*\d+-(\d+)'.encode()
```

Comment:
It's better to add note instead of deleting.
And add a note to describe why do you uncheck this point,such as # bzxxxx

Reply:
Good point and I will refine this here.